src/main/java/co/rsk/bitcoinj/script/RedeemScriptParserFactory.java

```diff
@@ -14,8 +14,8 @@ public static RedeemScriptParser get(List<ScriptChunk> chunks) {
         // Due to a validation error, during the time this federation existed in testnet
         // bitcoinj-thin would not detect it correctly as an ERP fed
         // We need to keep this behaviour for the given redeem script to keep the consensus in testnet
-        ScriptParserResult scriptParserResult = ScriptParser.parseScriptProgram(ERP_TESTNET_REDEEM_SCRIPT_BYTES);
-        if (scriptParserResult.getChunks().equals(chunks)) {
+        List<ScriptChunk> erpTestnetRedeemScriptChunks = ScriptParser.parseScriptProgram(ERP_TESTNET_REDEEM_SCRIPT_BYTES);
```

Comment:
```suggestion
        List<ScriptChunk> nonStandardErpRedeemScriptHardcodedChunks = ScriptParser.parseScriptProgram(NON_STANDARD_ERP_TESTNET_REDEEM_SCRIPT_HARDCODED_SERIALIZED);
```

wdyt? to be consistent with the parser name

Reply:
Updated!